docs/framework/data/adonet/sql/linq/downloading-sample-databases.md

```diff
@@ -1,48 +1,49 @@
-  
-### To download and install SQL Server Express 
-  
-1.  Go to the [SQL Server Express Editions](https://www.microsoft.com/sql-server/sql-server-editions-express) page.  
-  
-1.  Select **Download now**. 
-  
-1.  After the file has downloaded, double-click the file and follow the installation instructions in the setup program.  
-  
-## Download SQL Server Management Studio
-If you want to modify a database that you have downloaded, you can access the database from **Server Explorer** in the Visual Studio integrated development environment (IDE), or you can use Microsoft SQL Server Management Studio (SSMS).  
-  
-### To download SQL Server Management Studio  
+Download the Northwind sample database from the following page in the Microsoft Download Center:
+
+[Northwind and Pubs Sample Databases](https://go.microsoft.com/fwlink?linkid=64296)
+
+After the file has downloaded, double-click the file to extract the databases and scripts. By default, the files are installed in the folder `<drive>:\SQL Server 2000 Sample Databases`.
+
+Before you can use the Northwind database, you have to do one of the following things:
+
+- Recreate the database on an instance of SQL Server by running the `instnwnd.sql` script file in the installation folder.
+
+- Attach the `northwnd.mdf` file with its corresponding `*.ldf` log file to an instance of SQL Server.
+
+## <a name="get_sql"></a> Get SQL Server  
+
+SQL Server Express is a free, entry-level edition of SQL Server that you can redistribute with applications. Download SQL Server Express from the following page:
   
--   Follow the instructions at the [Download SQL Server Management Studio (SSMS)](/sql/ssms/download-sql-server-management-studio-ssms) page.  
+[SQL Server Express Editions](https://www.microsoft.com/sql-server/sql-server-editions-express)
+
+If you're using [Visual Studio](https://visualstudio.microsoft.com/free-developer-offers/), SQL Server Express LocalDB is included in the Community edition as well as the Professional and higher editions.  
+
+## <a name="get_ssms"></a> Get SQL Server Management Studio
+If you want to view or modify a database that you've downloaded, you can use SQL Server Management Studio (SSMS). Download SSMS from the following page:
+
+[Download SQL Server Management Studio (SSMS)](/sql/ssms/download-sql-server-management-studio-ssms) 
+
+You can also view and manage databases in the Visual Studio integrated development environment (IDE). In [Visual Studio](https://visualstudio.microsoft.com/free-developer-offers/), connect to the database from **SQL Server Object Explorer**, or create a Data Connection to the database in **Server Explorer**. Open these explorer panes from the **View** menu.
   
 ## See Also  
  [Getting Started](../../../../../../docs/framework/data/adonet/sql/linq/getting-started.md)
```

Comment:
we're switching the items on the see also section to use bullet lists for consistency (and because the way docs got migrated, it relies on having two blank spaces after each entry which is easy to break)

Reply:
Fixed in my latest commit.